health/notifications/alarm-notify.sh.in

```diff
@@ -1335,21 +1335,33 @@ send_telegram() {
 
   if [ "${SEND_TELEGRAM}" = "YES" ] && [ -n "${bottoken}" ] && [ -n "${chatids}" ] && [ -n "${message}" ]; then
     for chatid in ${chatids}; do
-      # https://core.telegram.org/bots/api#sendmessage
-      httpcode=$(docurl ${disableNotification} \
-        --data-urlencode "parse_mode=HTML" \
-        --data-urlencode "disable_web_page_preview=true" \
-        --data-urlencode "text=${emoji} ${message}" \
-        "https://api.telegram.org/bot${bottoken}/sendMessage?chat_id=${chatid}")
-
-      if [ "${httpcode}" = "200" ]; then
-        info "sent telegram notification for: ${host} ${chart}.${name} is ${status} to '${chatid}'"
-        sent=$((sent + 1))
-      elif [ "${httpcode}" = "401" ]; then
-        error "failed to send telegram notification for: ${host} ${chart}.${name} is ${status} to '${chatid}': Wrong bot token."
-      else
-        error "failed to send telegram notification for: ${host} ${chart}.${name} is ${status} to '${chatid}' with HTTP response status code ${httpcode}."
-      fi
+      successful=0
+      attempt_counter=0
+
+      while [ ${successful} -eq 0 ] && [ ${attempt_counter} -lt 60 ]; do
```

Comment:
Hi @MAH69IK !

While we are investigating the cause of the problem during shutdown, is it possible please to make the `60` configurable via `health_alarm_notify.conf`, and set it as default with 0 retries?

In other words, if someone does not change this value, to work as before with no retries ?

Thanks!

Reply:
I went forward and pushed a commit that makes the number of retries configurable and 0 by default. @thiagoftsm can you test it?